tests/measurements/test_state.py

```diff
@@ -857,6 +871,8 @@ def func():
         elif len(return_wire_order) == 3:
             i, j, k = return_wire_order
             exp_statevector = np.kron(np.kron(single_states[i], single_states[j]), single_states[k])
+        else:
+            raise ValueError("how did I get here?")
```

Comment:
silencing pylint.

Reply:
I feel that `assert` is typically more appropriate for branches that are never supposed to be reached.